types/index.d.ts

```diff
@@ -278,6 +278,11 @@ declare namespace RamdaAdjunct {
          */
         isNegative(val: any): val is number;
 
+        /**
+         * Checks if value is a negative zero (-0).
+         */
+        isNegativeZero(val: any): val is number;
```

Comment:
IMHO this is incorrect. 
```ts
isNegativeZero(val: any): boolean;
```
..would be more appropriate. This function can return `false` for number values.

Reply:
Totally agree. Tbh I'm not that well familiar with Typescript but seeing that now it totally makes sense!

Done ✅